### DISCUSSION TITLE
more font support

### DISCUSSION THREAD

**Discussion by takarajt** [rg35xx-cfw/rg35xx-cfw.github.io#155] [Q&A]
I have some games file that are name in Chinese character and file names appears in EmulationStation are showing ????? instead of the actual character.Therefore games can't be scrapped correctly. I had even chose the Chinese language in the menu didn't fix the issue. I had seen some places online said to overwrite the ttf file in /usr/share/fonts/truetype/nanum/NanumMyeongjo.ttf with a ttf that supported Chinese. Tried to use batocera-save-overlay doesn't appear to be able to fix the issue due to it will end up showing mount: can't setup loop device: Inappropriate ioctl for device. Anyone has a way to fix the issue? or will the next update have a fix for it? Thank you!!

**Comment by KingLJN** [comment]
In terms of character support, What if you try getting another EmulationStation theme package and swap out the font files with those with CJK character support, and alter the .xml files contained within to point to those font files you've chosen? 
Also, if you change the system language to to Chinese, chances are some settings might not be changed cleanly and it would be best if you reboot to see the full effect.

**Reply by takarajt** [reply]
Thanks, it worked.

**Comment by KingLJN** [comment]
![IMG_20240226_132824](https://github.com/rg35xx-cfw/rg35xx-cfw.github.io/assets/47855056/b48a9e47-1fa3-4228-9e7b-286a5d27d7f9)
